deploy/helms/event-collector/values.yaml

```diff
@@ -0,0 +1,38 @@
+rootDomain: k8s-nightly.epm.kharlamov.biz
```

Comment:
Do we really need to have these values here?

Reply:
this is default value, we can remove it